docs/tutorials/optimize/power/power_manage.md

```diff
@@ -1,3 +1,271 @@
+
+### Choosing and shutting down peripherals
+
+It might seem like an open door, but putting the MCU to sleep is only part of a low power design: Peripherals can draw much more power than the MCU. The LED in the beginning of the article is drawing ~2.8 mA, much more than the rest of the circuit in deep sleep. Therefore, make sure to choose components that fit your power budget and shut peripherals down when you don't use them. Radios often have sleep modes you can invoke, so make sure your drivers use these.
+
+Another option to consider is using a lower voltage design. Many MCUs can run at 1.8 V instead of 3.3 V, and choosing peripherals that can run on the same voltage drastically reduces your overall power consumption.
+
+## Troubleshooting
+
+### Stack overflow when enabling sleep tracing or CPU statistics
+
+When enabling sleep tracing or CPU stats, the idle thread has to allocate more data. On some devices, this leads to stack overflows on the idle thread. If you encounter this, you can increase the stack size of the idle thread by adding the following section under `target_overrides` in `mbed_app.json`:
+
+```
+        "rtos.idle-thread-stack-size": 1024
+```
+
+### Device not entering deep sleep even though tickless is enabled
+
+On some devices, the interrupt latency when running on the low-power tickers causes the device to drop bytes when running the serial at higher baud rates (such as 115,200). To mitigate this, these devices run tickless from the microsecond ticker instead of the low-power ticker, and this blocks deep sleep.
+
+If your application does not require high baud rates, you can set the following macro to re-enable tickless from the low-power ticker:
+
+```
+MBED_CONF_TARGET_TICKLESS_FROM_US_TICKER=0
+```
+
+### Device wakes up from deep sleep every second (or other period)
+
+Some devices wake from deep sleep for a small period every second, even when the device is instructed to wake up much later.
+
+<span class="images">![Blinky with 10 second interval on DISCO-L475VG-IOT01A1](../../../images/deepsleep-wakeup.png)<span>Blinky with 10 second interval on DISCO-L475VG-IOT01A1</span></span>
+
+This is related to the maximum timeout of the hardware low power ticker. It can only be asleep accurately for one second, so the device wakes up every second, and Mbed OS brings the device back into deep sleep afterward. You often can override this behavior by choosing a different clock source, but this is device specific. Look at the data sheet for your MCU. On the DISCO-L475VG-IOT01A1, you can override this by adding the following line to the `target_overrides` section of your `mbed_app.json`:
+
+```json
+        "target.lpticker_lptim_clock": 4
+```
+
+### Device does not sleep in bare-metal mode
+
+The sleep manager does not load when running Mbed OS in bare-metal mode. We may add this capability in a future release.
```

Comment:
we should add something like "you should manually call the `sleep()` function making sure you set up the wake up source`